x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecks.java

```diff
@@ -105,6 +104,15 @@ static DeprecationIssue oldIndicesCheck(IndexMetaData indexMetaData) {
         boolean hasDefaultMapping = indexMetaData.getMappings().containsKey(DEFAULT_MAPPING);
         int mappingCount = indexMetaData.getMappings().size();
         if (createdWith.before(Version.V_6_0_0)) {
+            if (".tasks".equals(indexMetaData.getIndex().getName())) {
+                return new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
+                    ".tasks index must be re-created",
+                    "https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html" +
+                        "#_indices_created_before_7_0",
```

Comment:
I think this link should be : https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking-changes-7.0.html#_indices_created_before_7_0

Reply:
Fixed, thanks.